samples/nrf9160/lwm2m_client/prj.conf

```diff
@@ -127,11 +127,6 @@ CONFIG_LWM2M_PORTFOLIO_OBJ_SUPPORT=y
 CONFIG_BASE64=y
 CONFIG_LWM2M_RW_SENML_JSON_SUPPORT=y
 CONFIG_LWM2M_RW_JSON_SUPPORT=y
-CONFIG_LWM2M_RW_CBOR_SUPPORT=y
 CONFIG_LWM2M_SECONDS_TO_UPDATE_EARLY=8
 CONFIG_LWM2M_COAP_MAX_MSG_SIZE=1024
 CONFIG_LWM2M_COMPOSITE_PATH_LIST_SIZE=10
-
-# Workaround for a missing cddl-gen Kconfig flag
```

Comment:
can you revert commit which add this in this PR. So it not cause any conflict if we cherry pick things to main branch